qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -98,6 +91,53 @@ def extract_parameter(block: ScaleBlock | ParametricBlock, config: Configuration
     return config.get_param_name(block)[0]
 
 
+def sympy_to_pyq(expr: sympy.Expr) -> ConcretizedCallable:
+    """Convert sympy expression to pyqtorch ConcretizedCallable object.
+
+    Args:
+        expr (sympy.Expr): sympy expression
+
+    Returns:
+        ConcretizedCallable: expression encoded as ConcretizedCallable
+    """
+
+    # base case - independent argument
+    if len(expr.args) == 0:
+        res = (
+            float(expr)
+            if str(expr).replace(".", "", 1).replace("-", "", 1).isdigit()
+            else str(expr)
```

Comment:
Would it make sense not to convert to a `str` here and use Sympy functionalities later ? Ie. when checking for rationality ?